public_html/css/sass/game.scss

```diff
@@ -0,0 +1,76 @@
+$total-height: 100%;
+
+#game-view, #canvas-view, #color-palette-view, 
+.total-area {
+  height: $total-height;
+}
+
+#canvas-view, #color-palette-view {
+  background-color: white;
+  border: 1px solid black;
+}
+
+.word-panel {
+  height: 10%;
+  position: relative;
+  text-align: center;
+  background-color: #E0EBEB;
```

Comment:
Color literals like `#E0EBEB` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E0EBEB` should be written as `#e0ebeb`